packages/interface-ipfs-core/src/refs-local.js

```diff
@@ -39,15 +40,45 @@ module.exports = (common, options) => {
 
       const imported = await all(importer(dirs, ipfs.block))
 
-      // otherwise go-ipfs doesn't show them in the local refs
-      await Promise.all(
-        imported.map(i => ipfs.pin.add(i.cid))
-      )
+      // rust-ipfs doesn't yet have pinning api, it'll just list all local cids
+      // in /refs/local
+      if (common.opts.type !== 'rust') {
```

Comment:
Per https://github.com/ipfs/js-ipfsd-ctl/compare/master...ipfs-rust:add_rust_ipfs_http#diff-d4165b1e3527a04dd7e9d370e18c2a1aR47 (also published as draft PR to js-ipfsd-ctl) which we are using in https://github.com/ipfs-rust/ipfs-rust-conformance